sci-electronics/drahnr-oregano/drahnr-oregano-0.84.39.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python3_{4,5,6} )
+PYTHON_REQ_USE='threads(+)'
+
+inherit gnome2-utils python-single-r1 waf-utils xdg-utils
+
+DESCRIPTION="Application for the schematic capturing and simulation of electrical circuits"
+HOMEPAGE="https://github.com/drahnr/oregano"
+SRC_URI="https://github.com/drahnr/oregano/tarball/v${PV} -> ${P}.tar.gz"
```

Comment:
`tarball` URIs are obsolete and do not generate stable files, please use `archive` URIs instead. Also please remember to remove the local distfile to make `repoman manifest` refetch the correct archive, and update all other relevant parts of ebuild.